### ISSUE TITLE
Question, Difference between APK's?

### ISSUE THREAD

**Issue by GlytchMeister** [tdex-network/tdex-app#598]
Hello, what is the difference between the different APK's in the releases?  I am assuming the Debug version is for developers, but what is the difference between signed and unsigned, in layman's terms?  Is unsigned the version where I would have to do something to independently verify the authenticity of the file and "sign" it myself?  From my understanding, that is an abundance of caution that is beyond my technical ability and threat model (not trying to evade law enforcement, just trying to preserve basic privacy and security), correct?

**Comment by tiero** [comment]
In your side both are the same app, just that the signed has the guarantee is not created by someone else
